caiman/source_extraction/cnmf/estimates.py

```diff
@@ -1162,6 +1162,62 @@ def manual_merge(self, components, params):
         if self.coordinates is not None:
             self.coordinates = caiman.utils.visualization.get_contours(self.A,\
                                 self.dims, thr_method='max', thr='0.2')
+            
+        self.evaluate_components(imgs, params, dview=None)
+        
+    def manual_add(self, components, imgs, imgs_residual, params):
```

Comment:
`imgs_residual` is defined but never used in this function. You can also create it directly from the state of the object, so no need to pass it.